Processor_CashFlows_SftpCommunication.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class Processor_CashFlows_SftpCommunication extends Jobs_SftpCommunication
+{
+    public function __construct($JobType, $FilenamePattern, $LogJob, $Verbose)
+    {
+        parent::__construct(
+            $JobType,
+            $FilenamePattern,
+            $LogJob,
+            Jobs_Chargeback_Cashflows::CASHFLOWS_SFTP_FILE_LIVESPAN,
+            $Verbose
+        );
+        $this->ProcessorName = 'CashFlows';
+        $this->setDoneDir(Jobs_Chargeback_Cashflows::DIR_TYPE_DONE);
+        $this->setCacheDir(Jobs_Chargeback_Cashflows::DIR_TYPE_CACHE);
+    }
+
+    public function setCommand($cmdDirectory)
```

Comment:
Looks like the `$Verbose` flag is being used as a way here to test the value of the `$command` variable is correct. 

It seems likely that having a function which constructs the `$command` string and returns it should be sufficient for this purpose if well tested. Once we have such a function and it a test, it may not even be necessary to echo the string out each time.